event_registration_cancel_reason/wizard/event_registration_cancel_log_reason.py

```diff
@@ -21,9 +21,9 @@ class EventRegistrationCancelLogReason(models.TransientModel):
     def default_get(self, var_fields):
         res = super(EventRegistrationCancelLogReason, self).default_get(
             var_fields)
-        registration = self.env['event.registration'].browse(
-            self.env.context['active_id'])
-        res['event_type_id'] = registration.event_id.type.id
+        registrations = self.env['event.registration'].browse(
+            self.env.context['active_ids'])
+        res['event_type_id'] = registrations.mapped("event_id.type").id
```

Comment:
No, your expression will raise a singleton error. But it's not only a code error, but there's no logic on cancelling registrations from multiple types. They don't have the same reasons.